### PR TITLE
Release September 22nd, 2022

### PR DESCRIPTION
**Pull request by mariahosfeld**
Includes

**New Features & Functions:**
#1518 Feature/profile donation link
#1556 Feature/planet cash dashboard section
#1563 New Message and Error for Delete Profile

**Fixes:**
#1553 Bugfix/complete signup
#1564 Makes the toggle Project Details/List button translatable
#1565 Fix Active button colour for all MUI Date Pickers 
#1571 Updates for Planet Cash
#1572 fix: removes text transform from MuiButton text
#1573 Cleanup for Sep release

**Other:**
#1567 Add beta tag for Bulk Codes submenu 
#1569 add "new" tag for profile donation link

### CODE REVIEW

**Comment by sagararyal** [comment]
#### 📝 Changed routes:


| [**/profile/donation-link**](https://planet-webapp-hyrorwole-planetapp.vercel.app/profile/donation-link) |
|:-:|
|<a href="https://planet-webapp-hyrorwole-planetapp.vercel.app/profile/donation-link">Screenshot of /profile/donation-link</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/q-fIxA69WkCWMWdnJgDPZ6tdrXZS12PCeh0RoKAnnMjmEVD8dJne1P_cWlJY4ddIBJDsi57xLSc_re_0RxOKkQ.png">(view full size)</a> |







Commit 80c0db9687a6872d0983d47fa1747d8713a0ced9 (https://planet-webapp-hyrorwole-planetapp.vercel.app).

**Comment by mariahosfeld** [comment]
Functionality looks good to me.

File: src/features/common/types/countries.d.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export interface CountryType {
+  code: string;
+  label: string;
+  phone: string;
+}
```
Comment (mohitb35) [inline src/features/common/types/countries.d.ts:5]: This is duplicate and should be removed

File: pages/profile/planetcash/index.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  PlanetCashTabs,
+} from '../../../src/features/user/PlanetCash';
+import i18next from '../../../i18n';
+
+const { useTranslation } = i18next;
+
+export default function PlanetCashPage(): ReactElement {
+  const { t, ready } = useTranslation('me');
+  const [progress, setProgress] = useState(0);
+
```
Comment (mohitb35) [inline pages/profile/planetcash/index.tsx:15]: Add the cleanup function added in the `/profile/planetcash/new` and `/profile/planetcash/transactions` here as well.

```
 // Cleanup function to reset state and address Warning: Can't perform a React state update on an unmounted component.
  useEffect(() => {
    return () => {
      setProgress(0);
    };
  }, []);
```

File: src/features/common/types/countries.d.ts (hunk 1 above)
Comment (mohitb35) [reply src/features/common/types/countries.d.ts:5]: Resolved in https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/1573

**Pull request merged**